LivyInterpreterIT fails by timeout while initializing cluster in {{setUp()}} method.

row log - https://api.travis-ci.org/jobs/201058130/log.txt?deansi=true
build config - https://travis-ci.org/apache/zeppelin/jobs/201058130/config

{code}
ERROR [2017-02-13 10:14:04,528] ({main} Logging.scala[error]:49) - Failed to initialize cluster.
org.scalatest.exceptions.TestFailedDueToTimeoutException: The code passed to eventually never returned normally. Attempted 87 times over 30.040869848000003 seconds. Last failure message: assertion failed: MiniHdfsMain hasn't started yet..
	at org.scalatest.concurrent.Eventually$class.tryTryAgain$1(Eventually.scala:420)
	at org.scalatest.concurrent.Eventually$class.eventually(Eventually.scala:438)
	at org.scalatest.concurrent.Eventually$.eventually(Eventually.scala:478)
	at org.scalatest.concurrent.Eventually$class.eventually(Eventually.scala:307)
	at org.scalatest.concurrent.Eventually$.eventually(Eventually.scala:478)
	at com.cloudera.livy.test.framework.MiniCluster.start(MiniCluster.scala:299)
	at com.cloudera.livy.test.framework.MiniCluster.deploy(MiniCluster.scala:230)
	at com.cloudera.livy.test.framework.Cluster$.liftedTree1$1(Cluster.scala:102)
	at com.cloudera.livy.test.framework.Cluster$.cluster$lzycompute(Cluster.scala:90)
	at com.cloudera.livy.test.framework.Cluster$.cluster(Cluster.scala:88)
	at com.cloudera.livy.test.framework.Cluster$.get(Cluster.scala:116)
	at org.apache.zeppelin.livy.LivyInterpreterIT.setUp(LivyInterpreterIT.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:264)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.AssertionError: assertion failed: MiniHdfsMain hasn't started yet.
	at scala.Predef$.assert(Predef.scala:179)
	at com.cloudera.livy.test.framework.MiniCluster$$anonfun$start$2.apply$mcV$sp(MiniCluster.scala:300)
	at com.cloudera.livy.test.framework.MiniCluster$$anonfun$start$2.apply(MiniCluster.scala:299)
	at com.cloudera.livy.test.framework.MiniCluster$$anonfun$start$2.apply(MiniCluster.scala:299)
	at org.scalatest.concurrent.Eventually$class.makeAValiantAttempt$1(Eventually.scala:394)
	at org.scalatest.concurrent.Eventually$class.tryTryAgain$1(Eventually.scala:408)
	... 27 more
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 30.525 sec <<< FAILURE! - in org.apache.zeppelin.livy.LivyInterpreterIT
{code}